Assets/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -84,6 +71,19 @@ public async Task SetUserDetailsAsync(CountlyUserDetailsModel userDetailsModel)
                     throw new Exception("Accepted picture formats are .png, .gif and .jpeg");
                 }
 
+                userDetailsModel.Name = TrimValue(userDetailsModel.Name);

```

Comment:
Every one of these should print a unique warning message if they are trimmed. That message should say which field was over the limit